varats-core/varats/experiment/wllvm.py

```diff
@@ -61,8 +62,39 @@ class RunWLLVM(base.Extension):  # type: ignore
     This class is an extension that implements the WLLVM compiler with the
     required flags LLVM_COMPILER=clang and LLVM_OUTPUFILE=<path>. This compiler
     is used to transfer the complete project into LLVM-IR.
+
+    The distcc_hosts argument will instruct WLLVM to call distcc instead of
+    clang directly. It is the user's responsibility to ensure that all
+    distcc hosts are configured to use exactly the same compiler as localhost!
     """
 
+    def __init__(
+        self,
+        distcc_hosts: tp.Optional[str] = None,
+        *extensions: 'Extension',
+        config: tp.Optional[tp.Dict[str, str]] = None,
+        **kwargs: tp.Any
+    ):
+        super().__init__(*extensions, config=config, **kwargs)
+        self._distcc_hosts = distcc_hosts
+
+    def __create_distcc_scripts(self) -> tp.Tuple[Path, Path]:
+        projdir = Path(sys.argv[0]).parent
```

Comment:
```suggestion
        project_dir = Path(sys.argv[0]).parent
```
This is the more common spelling in the code base

Reply:
Fixed in 3e5d671